src/elf.rs

```diff
@@ -301,6 +303,19 @@ impl<E: UserDefinedError, I: InstructionMeter> Executable<E, I> for EBpfElf<E, I
                             .get_at(sym.st_name)
                             .ok_or(ElfError::UnknownSymbol(sym.st_name))?;
                     }
+                } else if let Some(BpfRelocationType::R_Bpf_64_64) =
```

Comment:
How about switching to `match`?
`match BpfRelocationType::from_x86_relocation_type(relocation.r_type) {`

Reply:
updated